src/selections-factory.js

```diff
@@ -81,5 +77,22 @@ export default function initSelections() {
     };
   }, [api]);
 
+  useEffect(() => {
+    const onClick = (e) => {
+      const classes = e.target.className;
+      // TODO: isSelectableCell is false when dragging from one cell to another
+      const isSelectableCell = classes.includes('sn-table-cell') && !classes.includes('sn-table-excluded');
+      if (selections.api.isActive() && !isSelectableCell) {
+        e.stopPropagation();
+        selections.api.confirm();
+      }
+    };
+
+    el.addEventListener('click', onClick, true);
+    return () => {
+      el.removeEventListener('click', onClick, true);
+    };
```

Comment:
Coverage here is a problem here... Need to figure out a way to test this logic.

Reply:
You can add RTL tests that tests logic inside hooks

Reply:
Checkout out `renderHook` from RTL

Reply:
Nice, setting up RTL is on the todo list 👍